contracts/access/manager/AccessManager.sol

```diff
@@ -109,8 +109,7 @@ contract AccessManager is Context, Multicall, IAccessManager {
     bytes32 private _executionId;
 
     /**
-     * @dev Check that the caller is authorized to perform the operation, following the restrictions encoded in
-     * {_getAdminRestrictions}.
+     * @dev Check that the caller is authorized to perform the operation.
```

Comment:
Maybe the comment can be kept but we can change brackets for `` ? Private references should be treated differently.

Reply:
It's true private references should be treated differently, although in this case I think a reference to where the rules are defined is more important than the reference itself.

Let's do:

```suggestion
     * @dev Check that the caller is authorized to perform the operation.
     * See {AccessManager} description for a detailed breakdown of the authorization logic.
```